build.zig

```diff
@@ -92,7 +92,12 @@ pub fn build(b: *std.Build) void {
     });
     builtins_tests.root_module.stack_check = false;
 
-    if (!no_bin) {
+    b.default_step.dependOn(&all_tests.step);
+    b.default_step.dependOn(&builtins_tests.step);
```

Comment:
Note: this leads to `zig build` compiling, but not running tests, and `zig build -Dno-bin` checking that all tests compile.